apps/frontend/src/libs/components/app-header/app-header.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useMemo, useRef, useState } from "react";
+
+import {
+	arrowDown,
+	logo,
+	profileDefault,
+} from "~/assets/img/header/header.img.js";
+import { Link } from "~/libs/components/components.js";
+import { useAppSelector, useDropdownMenu } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+import { UserMenu } from "./user-menu.js";
+
+const AppHeader: React.FC = () => {
```

Comment:
Pay attention that the header should appear only on certain pages according to Figma